modules/certs/pages/index.adoc

```diff
@@ -0,0 +1,159 @@
+// Type: procedure
+[id="installing-cli"] 
+= Creating sites using a custom certificate authority on OpenShift
+
+By default, {skupper-name} creates certificates to establish links between sites using mutual TLS.
+These certificates are stored as secrets in the namespace when you create a site using `skupper init`.
+If you want to use your own certificates, you can populate the a set of secrets with the appropriate certificates before creating the site as described in this section.
+This set of secrets provides {skupper-name} with the configuration required to create a site.
+
+The following certificates are required:
```

Comment:
@nluaces  I added more to the introduction (from gdoc), but I reduced it considerably, still ok?

Reply:
It's good, thank you